src/polyfill/chardetng.js

```diff
@@ -0,0 +1,38 @@
+goog.module('polyfill.chardetng');
+
+const TrustedResourceUrl = goog.require('goog.html.TrustedResourceUrl');
+const {createConstant} = goog.require('os.string');
+const {safeLoad} = goog.require('goog.net.jsloader');
+const os = goog.require('os');
+
+/**
+ * @define {string} Base path to the jschardet library from the OpenSphere root.
+ */
+const JSCHARDET_BASE_PATH = goog.define('JSCHARDET_BASE_PATH', 'vendor/jschardet');
```

Comment:
This covers the compiled build, but we'll want to add it to `moduleDefines` in `package.json` for the dev build.